frontend/src/scenes/hog-functions/logs/HogFunctionLogs.tsx

```diff
@@ -153,26 +157,30 @@ function HogFunctionLogsStatus({
 
             <LemonMenu
                 items={[
-                    eventId
+                    eventId && !internalEvent
                         ? {
                               label: 'View event',
                               to: urls.event(eventId, ''),
                           }
                         : null,
-                    {
-                        label: 'Retry event',
-                        disabledReason: !eventId ? 'Could not find the source event' : undefined,
-                        onClick: () => retryInvocations([record]),
-                    },
-                    {
-                        label: 'Select for retry',
-                        onClick: () => {
-                            setSelectingMany(true)
-                            setSelectedForRetry({
-                                [record.instanceId]: true,
-                            })
-                        },
-                    },
+                    internalEvent
+                        ? null
+                        : {
+                              label: 'Retry event',
+                              disabledReason: !eventId ? 'Could not find the source event' : undefined,
+                              onClick: () => retryInvocations([record]),
+                          },
+                    internalEvent
+                        ? null
+                        : {
+                              label: 'Select for retry',
+                              onClick: () => {
+                                  setSelectingMany(true)
+                                  setSelectedForRetry({
+                                      [record.instanceId]: true,
+                                  })
+                              },
+                          },
                     {
```

Comment:
I guess this also doesn't work if you are hiding the testing tab?

maybe we should just hide this whole menu?

Reply:
Yeah but it wont work as there are no events to load from clickhouse 😅

Reply:
Like this whole thing (like the Testing tab) is dependent on loading real events from clickhouse, whereas these are all these "meta" events that dont exist anywhere but the stream

Reply:
Like if you can't "view" the event then you also cant load it into the testing thing.

The logs dont contain the event, only the uuid (which in this case isn't a _clickhouse_ event uuid, just the ephemeral one

Reply:
Gotcha - thanks for the explanation. Updated to remove the menu entirely